DOCSUP-921 (#113)
BayoNet/ClickHouse: docs/en/operations/system-tables.md

Each database that the server knows about has a corresponding entry in the table.
This system table is used for implementing the `SHOW DATABASES` query.

## system.data_type-families {#system_tables-data_type_families}

> Contains information about supported ([data types](../sql-reference/data-types/)).

Reviewer: ```suggestion
Contains information about supported [data types](../sql-reference/data-types/index.md).
```

Each database that the server knows about has a corresponding entry in the table.
This system table is used for implementing the `SHOW DATABASES` query.

> ## system.data\_type-families {#system_tables-data_type_families}

Reviewer: ```suggestion
## system.data_type_families {#system_tables-data_type_families}
```

> Contains information about properties of SQL data types.

Reviewer: ```suggestion
Contains information about suported data types.
```

Reviewer: Here must be a link to data types introduction.

Columns:

-   `name` ([String](../sql-reference/data-types/string.md)) — Data type name.

> -   `case_insensitive` ([UInt8](../sql-reference/data-types/int-uint.md)) — Shows whether a data type is case sensetive.

Reviewer: ```suggestion
-   `case_insensitive` ([UInt8](../sql-reference/data-types/int-uint.md)) — Property that shows whether you can use a data type name in a query in case insensitive manner or not. For example, `Date` and `date` are valid both.
```

> -   `alias_to` ([String](../sql-reference/data-types/string.md)) — Equivalents to data types of C.

Reviewer: ```suggestion
-   `alias_to` ([String](../sql-reference/data-types/string.md)) — Data type name for which `name` is an alias.
```

**Example**

``` sql
SELECT * FROM system.data_type_families 
```

> ``` text

Reviewer: Too long output. I'd use more interesting example `SELECT * FROM system.data_type_families WHERE alias_to = 'String'` with shorter output.
